sdk/storage/storage-blob/CHANGELOG.md

```diff
@@ -18,6 +18,10 @@
 - Added support for listing system containers with `BlobServiceClient.listContainers()`.
 - Added support for blob names container invalid XML characters.
 
+### Other changes
```

Comment:
12.9.0-beta.2 has been released. The changed log should be add to `## 12.9.0-beta.3 (Unreleased)`.